### PR TITLE
Todo List : Add & Remove 

### PR DESCRIPTION
**Pull request by jeanbuhendwa**
# Todo List 

### In this pull request, I have: 

- [ ] Removed all hardcoded items from the tasks array.
- [ ] Created a new JavaScript file for the new functionality.
- [ ] Implemented a function for adding a new task (add a new element to the array).
- [ ] Implemented a function for deleting a task (remove an element from the array).
- [ ] Implemented a function for editing task descriptions.
- [ ] Saved all changes to the To-Do List in local storage.

### CODE REVIEW

File: src/modules/task.js [hunk 1]
```diff
@@ -1,46 +1,144 @@
+// const resetBtn = document.getElementById('reset-all');
```
Comment (GedeonTS) [inline src/modules/task.js]: - Kindly make sure to remove this commented out line of code to make your code cleaner :recycle:

File: src/modules/task.js [hunk 2]
```diff
@@ -1,46 +1,144 @@
...
+  localStorage.setItem('Todo', JSON.stringify(listArr));
+});
+
+addBtn.addEventListener('click', () => {
+  addTask();
+});
+
+// resetBtn.addEventListener("click", () => {
+//   localStorage.clear();
+// });
```
Comment (GedeonTS) [inline src/modules/task.js]: - Kindly make sure to remove this commented out lines of code to too, make your code cleaner :recycle:

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :

Hi @jeanbuhendwa , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :tophat:  :+1: 
- Creating functionality is working well :white_check_mark: 
- Editing functionality is working well :white_check_mark: 
- Removing functionality is working well :white_check_mark: 
- No linter errors :+1: 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @jeanbuhendwa 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- Removed commented code 👏🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**